### ISSUE TITLE
map_result_to_celltypes for B cell not working for cibersort(_abs)

### ISSUE THREAD

**Issue by mfilip8** [icbi-lab/immunedeconv#31]
Hi all,
thanks a lot for the great package and work you are providing. 
I have found that when I am trying to map results for cibersort(_abs) to B cell fractions it does not work and give NA as result: 
map_result_to_celltypes(res_cibersort_abs, "B cell", "cibersort_abs")

For all other cell types (e.g. cell_types = c("T cell CD4+", "T cell CD8+", "NK cell", "Macrophage M1", "Macrophage M2", "Monocyte", "T cell regulatory (Tregs)", "Myeloid dendritic cell", "Neutrophil")) it works perfectly.
I can work around it but it will be great to have it fixed for automatisation.

Thanks again in advance
Marco

**Comment by grst** [comment]
Hi Marco, 

thanks for reporting. 
This is for sure a problem in the cell-type mapping table. I'll investigate. 

Cheers, 
G

**Comment by grst** [comment]
This should be fixed by PR #32.
Can you confirm that it works before I merge it? 
 
You can test it by installing from the `fix/celltypemapping` branch: 
```R
remotes::install_github("icbi-lab/immunedeconv", ref="fix/celltypemapping")
```
